frontend/lib/search/search_page.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:ehrenamtskarte/search/result_list.dart';
+import 'package:flutter/material.dart';
+
+import 'filter_bar.dart';
+
+class SearchPage extends StatefulWidget {
+  @override
+  createState() => _SearchPageState();
+}
+
+class _SearchPageState extends State<SearchPage> {
+  String _searchFieldText;
+  TextEditingController _textEditingController;
+  List<int> _selectedCategories;
+
+  @override
+  Widget build(BuildContext context) {
+    return CustomScrollView(
+      slivers: [
+        SliverAppBar(
+          title: TextField(
+            onChanged: (text) {
+              setState(() {
+                _searchFieldText = text;
+              });
+            },
+            controller: _textEditingController,
+            decoration: InputDecoration.collapsed(
+              hintText: "Tippen und schreiben, um zu suchen …",
+            ),
+          ),
+          pinned: true,
+          actions: [
+            IconButton(
+              icon: const Icon(Icons.search),
+              onPressed: () => {},
+            )
+          ],
```

Comment:
```suggestion
```
I think we can leave away the button if it does nothing anyway.

Reply:
Now, it does focus the text field :)